CDCgov/NEDSS-Modernization
# CNFT2-946-save-concept

## Description

While clicking on save changes link on Value set concepts Tab we need a API to save Concepts to database.

## Tickets

* [CNFT2-946](https://cdc-nbs.atlassian.net/browse/CNFT2-946)

## Steps to verify

1. You can use the swagger api endpoints to create update requests, and then check the classic UI to see the changes you have made and what affect it has within the UI
2. Another way to do this is to manually look at the database on your local to see the changes

[CNFT2-946]: https://cdc-nbs.atlassian.net/browse/CNFT2-946?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
